kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/NetworkService.java

```diff
@@ -82,7 +82,31 @@
      */
     public List<NetInterface<? extends NetInterfaceAddress>> getActiveNetworkInterfaces() throws KuraException;
 
-    public String getModemUsbPort(String interfaceName);
+    /**
+     * Given an interface name (e.g. 'ppp0'), look up the associated usb port
+     * 
+     * @param the
+     *            name of the ppp interface (i.e. ppp0)
+     * @return a string representing the usb port of the modem (i.e. 1-2.3)
+     */
+    public String getModemUsbPort(String pppInterfaceName);
 
+    /**
+     * Given a modem device, look up the associated ppp interface name
+     * 
+     * @param modemDevice
+     * @return the name of the ppp interface
+     * @throws KuraException
```

Comment:
Missing since annotation

Reply:
I added the since annotation to this method only, since the two methods above are there from the first Kura contribution. I've only added the javadocs.